deployer/commands/deployer.py

```diff
@@ -68,6 +70,21 @@ def deploy_support(
             )
 
 
+@contextmanager
+def custom_chart(chart_dir, chart_override_path):
+    try:
+        if chart_override_path:
+            chart_yaml = chart_dir / "Chart.yaml"
+            temp_default_chart_copy = chart_dir / "Chart-copy.yaml"
+
+            shutil.copy(chart_yaml, temp_default_chart_copy)
+            shutil.copy(chart_override_path, chart_yaml)
+        yield chart_dir
+    finally:
+        if chart_override_path:
+            os.replace(temp_default_chart_copy, chart_yaml)
```

Comment:
Whenever we run a helm command, we can run it under this context manager that:

- makes a copy of the current Chart.yaml
- copies the hub's custom chart in the basehub's Chart.yaml

When it's done, it replaces Chart.yaml with the original file and removes the copy.

Reply:
Ah, correct. It's funny, cause this was the first idea I had, then replacing Chart.yaml felt cleaner and more straightforward.